### PR TITLE
Talk styling update

### LINKED ISSUE #629: if I click the most recent talk post link it brings me to the post but in Zooniverse wide talk
related to the previous issue maybe of the navigation side bar

If I click on the link to the most recent talk discussion (We're in a beta test for my project so there are some talk comments), it takes you the message but everything including the address changes to Zooniverse wide talk 

screen shot 2015-06-20 at 2 42 43 pm

### LINKED ISSUE #716: Displaying username along with the Display Name on Talk
I noticed that I can add a display name, but then realized if someone wanted to @ me on Talk , they would need to use my username, but there's no place to tell what my username is other than clicking on my profile. It seems like a simple solution would be to add the username along with the display name even if its a temp solution.

### LINKED ISSUE #727: click on a hashtag on a project Talk side bar and it searchs Zooniverse Talk
Screen that shows after I click dustdevil on the screenshot below this one
screen shot 2015-06-25 at 2 17 37 am

screen shot 2015-06-25 at 2 18 02 am

### PR DESCRIPTION
**Pull request by aweiksnar**
- About half-way through getting the talk css in place, but figured I'd PR early since this closes a few other first release issues
- Closes #629
- Closes #727
- Closes #716

### CODE REVIEW

**Comment by edpaget** [comment]
Cool I'm just going to post things as I see them in the UI. 

It seems like the Discussion Titles shouldn't be underlined: 
screenshot from 2015-06-24 15 07 29

**Comment by edpaget** [comment]
Board title link needs a :hover state

screenshot from 2015-06-24 15 09 57

**Comment by edpaget** [comment]
Username/@mention/roles could be a bit bigger
screenshot from 2015-06-24 15 12 08

**Comment by edpaget** [comment]
Though take that one with a grain of salt since I have a 1080p screen that's only 13" without any scaling on.

**Comment by edpaget** [comment]
It'd be cool for the username to also have a hover state on the boards page.

**Comment by edpaget** [comment]
I think it looks good far :+1:

File: app/collections/favorites-button.cjsx [hunk 1]
```diff
@@ -89,6 +89,9 @@ module?.exports = React.createClass
         className="favorites-button"
         type="button"
         onClick={@toggleFavorite}>
-        <i className={"fa fa-heart#{if favorited then '' else '-o'}"} />
+        <i className="
+          fa fa-heart#{if @state.favorited then '' else '-o'}
```
Comment (edpaget) [inline app/collections/favorites-button.cjsx]: This isn't set on state anymore just from the PromiseRenderer. Can you also delete it from the getInitialState: -> function?

Comment (aweiksnar) [inline app/collections/favorites-button.cjsx]: Ah sorry I should've checked that more closely in the merge, I'll push that up with the other changes in one sec

**Comment by aweiksnar** [comment]
Awesome thanks for the feedback - I think I got everything in a4935f73bd4edce89efd7a3e20ad758613d1d25f

**Pull request merged**